### PR TITLE
Week 6: Survey - Alma and Linda

### PR DESCRIPTION
**Pull request by almaherris**
## Netlify link
https://surprise-holiday.netlify.app/

## Collaborators
[almaherris, linda-f]

### CODE REVIEW

**Review by AntonellaMorittu — APPROVED** [review]
Hello and congrats with your Survey project! The app satisfies the requirements and looks also good. Well done with responsiveness and accessibility, with code format and project structure, you nailed it! 👏

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+.name-input,
+.destination-input {
+  padding: 10px 5px;
+  border-radius: 6px;
+  border: 1px solid black;
+  font-size: 18px;
+}
+
+.name-input:focus::placeholder {
+  color: transparent;
```
Comment (Martin-Joensson) [inline src/App.css:52]: What does this do? Is it when you tab to the name input field?

File: src/App.jsx [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
-};
+  const [showSummary, setShowSummary] = useState(false)
+  const [formData, setFormData] = useState({
+    name: "",
+    destinationType: "",
+    period: "",
+    budget: "",
```
Comment (Martin-Joensson) [inline src/App.jsx:20]: Very nicely formatted state for the form data.

File: src/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import "./Header.css"
+
+export const Header = () => {
+  return (
+    <header className="header">
+      <h1 onClick={() => location.reload()}>Surprise Holiday</h1>
```
Comment (Martin-Joensson) [inline src/Header.jsx:6]: nice little extra functionallity with the header.

**Review by Martin-Joensson — COMMENTED** [review]
Very nice project guys! I especially like the range slider to set your budget. =) 

/Martin

**Pull request closed without merging**